backend/migrations/versions/bd271a2780b2_init_database.py

```diff
@@ -1253,18 +1253,6 @@ def upgrade():
                 sa.Column('created', sa.DateTime(), nullable=True),
                 sa.PrimaryKeyConstraint('AthenaQueryId'),
             )
-            op.create_table(
```

Comment:
We cannot modify pre-existing alembic migration scripts. If we are going to delete a table we need to create a new script deleting it otherwise nothing gets deleted in the pre-existing deployments

Reply:
In a new deployment, all migration scripts are run in order, so the changes will work. But let's say we have a deployment in V1.3, when we upgrade the CodeBuild dbmigration stage runs `alembic upgrade` (or something similar) and executes the new migration scripts only

Reply:
as we discussed, I removed the unnecessary changes from the former migration scripts, and fixed the changes in the [latest one](https://github.com/awslabs/aws-dataall/pull/449/files#diff-885ec94a722c02b6bbdedc29b9e758a8076b9517ff6d96f856ee38a41fac6634)

Reply:
Looks good, I will review the comments in mlstudio and see if I have time to test this one